frontend/src/stories/components/ArchiveSearch/ArchiveSearchForm.stories.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+
+import ArchiveSearchForm from "main/components/ArchiveSearch/ArchiveSearchForm";
+import {allTheSubjects} from "fixtures/subjectFixtures";
+import { systemInfoFixtures } from "fixtures/systemInfoFixtures";
+
+export default {
+    title: "components/ArchiveSearch/ArchiveSearchForm",
+    component: ArchiveSearchForm,
+    parameters: {
+        data: [
```

Comment:
Change this line to mockData and I think you'll find that the subjects start showing up.

See item 3 here: 
https://storybook-addon-mock.vercel.app/?path=/story/docs-installation--page